src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -286,21 +284,23 @@
                                (data "jobId")])
                 (let [status (data "executionStatus")]
                   (create-field "Status" (moncommon/icon-for-call-status status) status))
-                (when (= (get-in data ["callCaching" "effectiveCallCachingMode"]) "ReadAndWriteCache")
+                (when (and (:use-call-cache props) (not (data "subWorkflowId")))
                   (create-field "Cache Result" (moncommon/format-call-cache (get (data "callCaching") "hit"))))
                 (create-field "Started" (moncommon/render-date (data "start")))
                 ;(utils/cljslog data)
                 (create-field "Ended" (moncommon/render-date (data "end")))
                 [IODetail {:label "Inputs" :data (data "inputs") :call-detail? true}]
                 [IODetail {:label "Outputs" :data (data "outputs") :call-detail? true}]
-                (create-field "stdout" (display-value (data "stdout") (last (string/split (data "stdout") #"/"))))
-                (create-field "stderr" (display-value (data "stderr") (last (string/split (data "stderr") #"/"))))
+                (when (not (data "subWorkflowId"))
+                  (do
+                    (create-field "stdout" (display-value (data "stdout") (last (string/split (data "stdout") #"/"))))
```

Comment:
I'd prefer a more generic "when data exists" like the `when-let`s around 315

Reply:
oh you did that on the next commit :)

Then I want to update this by saying to use when-let so we don't need to reference the data twice